src/JWTAuth.php

```diff
@@ -57,11 +57,13 @@ public function authenticate()
     {
         $id = $this->getPayload()->get('sub');
 
-        if (! $this->auth->byId($id)) {
-            return false;
+        $retrieveUser = config("jwt.retrieve_user_on_authentication", true);
```

Comment:
It might sound strange, given that this package is aimed at LaraveI, but I don't really want to bake in the config function here.

You'll see that i've taken care not to do things like this throughout the package, and i've delegated all of the config to the service providers where it belongs :)